starwars/public/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
Hi Brent,
./src/App.js
Module not found: Can't resolve './components/Films' in 'C:\Teaching\Spring 2018\week5\starwars\src'
import TheFilms from './components/Films'
was there supposed to be a json file for Films?

Reply:
it's pulling from the API, _I didn't have all the files pushed my apologies_, please see the [latest commit](https://github.com/modern-web-application-uw18/week5/pull/18/commits/e79b1cf720f265ecc89bd288c3cba16a39377a1a) .

ThePeople & TheFilms are both files under the component folder.